RFC/readme.md

```diff
@@ -0,0 +1,142 @@
+discussions.
+
+Here follows the RFC structure.
+
+1. Request for Comments (RFCs) for Dgraph Core products. It will be located at "Proposals".
+2. RFC for deprecation of code components. It will be located at "Deprecated".
+3. Post-mortem analysis for resolved RFCs and deprecation processes. It will be located at "Post-mortem".
+
+The RFC process provides a structured way to discuss and evaluate potential changes, while the deprecation process allows
+us to provide an estimated timeline (ETA) for determining whether to continue with the process or cancel it based on
+community interest. The Post-mortem directory will allow us to document and analyze the outcomes of resolved RFCs and
+deprecation processes, which will help us improve our decision-making process in the future.
+
+### How to start
+
+The creation of an RFC in the Dgraph Core can start from a variety of sources. For example, an idea may arise from a
+discussion on the Discourse forum or other community channels. If the proposal is considered to be valid, and contains a
+logical or technical solution that can be implemented, it can be submitted as an RFC.
+
+It is not necessary for the person submitting the proposal to discuss it beforehand. If they have sufficient knowledge
+to propose an idea and the technical skills to understand the Core code, they can open a proposal. However, it is
+encouraged to engage in discussion with the community to validate the idea and refine the proposal.
+
+Once the proposal is submitted, it will go through a review process where the community and the core development team
+can provide feedback and suggestions. The proposal may be revised and updated based on this feedback. Once the proposal
+is deemed ready, it can be merged into the Dgraph Core codebase.
+
+
+## Organizing RFCs
+
+When contributing to the Dgraph RFC repository, it's important to follow a clear and organized structure to ensure that
+all proposals and discussions are easily accessible and understandable.
+
+Each Dgraph product has its own specific directory, such as "Dgraph_Core", "BadgerDB_Core", and so on. In the top level
+of these directories, there will be the "Proposals", "Deprecated", and "Post-mortem" directories.
+
+When submitting a new RFC, you should create a new directory within the relevant product directory. For example, if you
+are submitting an RFC for Dgraph Core, you should create a directory named "RFC-D0000" (where the numbers can be
+adjusted to suit your proposal). You may choose to create your RFC in either .MD or .mediawiki format.
+
+Additionally, you could include any supporting files, such as images or videos, within the same directory as your RFC.
+
+Here's an overview of the file structure:
+
+```lua
+Dgraph RFC Repository
+|
+|-- Proposals
+|   |
+|   |-- Dgraph_Core
+|   |   |
+|   |   |-- RFC-D0000
+|   |   |   |
+|   |   |   |-- readme.md
+|   |   |   |
+|   |   |   |-- Supporting files (e.g. images, videos)
+|   |
+|   |-- BadgerDB_Core
```

Comment:
As discussed, please remove the BadgerDB_Core directory here. This should be move to the badger repo.